script/build.ts

```diff
@@ -7,9 +7,11 @@ const root = dirname(__dirname)
 function main() {
   const zmq_rev =
     // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing, @typescript-eslint/strict-boolean-expressions
-    process.env.ZMQ_VERSION || "20de92ac0a2b2b9a1869782a429df68f93c3625e"
-  const src_url = `https://github.com/zeromq/libzmq/archive/${zmq_rev}.tar.gz`
+    process.env.ZMQ_VERSION || "4.3.5"
 
+  // if it looks like a dot-separated version number, prepend with "v" to match repo tagging convention
+  const gitref = zmq_rev.match(/^\d+\./) ? `v${zmq_rev}` : zmq_rev
+  const src_url = `https://github.com/zeromq/libzmq/archive/${gitref}.tar.gz`
```

Comment:
I use an exact commit hash instead of tags because upstream is slow in tagging their releases.

Reply:
That makes a ton of sense and I love having the ability to have a hash. The version number is much more friendly. These changes still *allow* for a hash. I'll add a comment.